src/locutus_util/_version.py

```diff
@@ -28,7 +28,7 @@
 commit_id: COMMIT_ID
 __commit_id__: COMMIT_ID
 
-__version__ = version = '2.0.1.dev19+g5d43c2f60.d20250903'
-__version_tuple__ = version_tuple = (2, 0, 1, 'dev19', 'g5d43c2f60.d20250903')
+__version__ = version = '2.0.1.dev28+g9f8ea4469.d20251103'
```

Comment:
I wonder if we should bump this up to 2.0.2. I suspect that would involve a new tag